docs/semgrep-assistant/overview.md

```diff
@@ -23,11 +23,11 @@ Semgrep Assistant provides GPT-4-powered security recommendations to help you re
 
 ## Support and availability
 
-Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets. It requires Semgrep Cloud Platform and is not available on the Semgrep CLI.
+Semgrep Assistant primarily supports findings generated by Semgrep Code. It requires Semgrep Cloud Platform.
 
 Semgrep Assistant is available to users of the following source code managers (SCMs):
   * GitHub Cloud
-  * GitLab, including SaaS, self-managed, and self-hosted plans
+  * GitLab, including SaaS and self-managed plans
```

Comment:
As far as I know, "self-managed" is GitLab's term for self-hosted. They do also have something called "Dedicated" which I'm not particularly familiar with.

Reply:
Yes, "self-managed" is the term used by GitLab, so I removed self-hosted for that reason -- do you think that's confusing?